documentation/modules/ref-jvm-options.adoc

```diff
@@ -122,3 +122,18 @@ jvmOptions:
   gcLoggingEnabled: true
 # ...
 ----
+
+=== Output Garbage collector logs to a file
+
+The `jvmOptions` section can also be configured to output the garbage collector (GC) logging to a file.
+Once GC logging has been enabled (see above) you can direct the GC logs to a file.
+To enable it, set the `Xloggc` property as follows:
+
+.Example of enabling output of GC logs to a file
+[source,yaml,subs=attributes+]
+----
+# ...
+jvmOptions:
+  "-Xloggc": "/tmp/gc.log"
```

Comment:
How does this combine with `gcLoggingEnabled`? Does that need to be enabled? Or will it be enabled automatically? Or is that STDOUT only?

Reply:
`gcLoggingEnabled` should be set to true. This is highlighted in the text prior to the example but will add it to the code example to make it clearer 👍

Reply:
👍 I think it needs to be in the example so that people can copy paste this.